data/35e/wizards_of_the_coast/supplement/complete_adventurer/cad_skills.lst

```diff
@@ -1,3 +1,3 @@
 
 
-Craft (Poisonmaking)										KEYSTAT:INT	USEUNTRAINED:NO						TYPE:Intelligence.Craft.Standard.Base.Craft (Poisonmaking)													SOURCEPAGE:p.97
+Craft (Poisonmaking)				KEYSTAT:INT	USEUNTRAINED:NO				TYPE:Intelligence.Craft.Standard.Base	VISIBLE:DISPLAY			SOURCEPAGE:p.97
```

Comment:
While I don't understand why it's happening the way that it is (as I've tried updating the data a few different ways), the display of the Craft (Poisonmaking) in 6.04 and 6.06.00RC2 both display Craft (Poisonmaking) on a GUI tree node/left peer level to Craft - in fact, Craft (Poisonmaking) appears in the tree as Craft.Craft (Poisonmaking).

As for the VISIBLE tag, that was me mimicking the WotC SRD data file in a vain attempt to get it to display correctly.

Please see the attached images of the "before fix" and "after fix" for a visual representation of the problem from the 6.06.00RC2 release.

![craft_before](https://cloud.githubusercontent.com/assets/14807152/13100187/cea8363c-d4ff-11e5-90f5-133c2fd8620e.jpg)

![craft_after](https://cloud.githubusercontent.com/assets/14807152/13100189/d3b54782-d4ff-11e5-8cb1-b8cf9e4ed015.jpg)